apps/src/gamelab/animationListModule.js

```diff
@@ -315,6 +373,19 @@ export function addAnimation(key, props) {
 }
 
 /**
+ * Append an animation to the project (at the end of the list of frames).
+ * @param {!SerializedAnimation} props
+ */
+export function addCustomAnimationAddition(props) {
+  return (dispatch, getState) => {
+    var selectedAnimationKey = getState().animationTab.selectedAnimation;
+    dispatch(setPendingAnimationAddition(selectedAnimationKey, props));
+    dispatch(loadAnimationAdditionFromSource(selectedAnimationKey, props));
+    dashboard.project.projectChanged();
```

Comment:
I suspect we could directly import `projectChanged` from project.js instead of using `dashboard.project.projectChanged()`? (here and elsewhere)